packages/contract-wrappers/src/contract_wrappers.ts

```diff
@@ -161,9 +115,8 @@ export class ContractWrappers {
      */
     public unsubscribeAll(): void {
         this.exchange.unsubscribeAll();
-        this.erc20Token.unsubscribeAll();
-        this.erc721Token.unsubscribeAll();
-        this.etherToken.unsubscribeAll();
```

Comment:
Is the WETH contract not included?

Reply:
The `etherTokenWrapper` looked like a wrapper around potentially many WETH9 tokens. Users had to pass in the token address for each method call.

```
_etherTokenContractsByAddress: {
        [address: string]: WETH9Contract;
    } = {};

...
public async depositAsync(
        etherTokenAddress: string,
        amountInWei: BigNumber,
        depositor: string,
        txOpts: TransactionOpts = {},
    ): Promise<string> {
...
```

Should we change it to work with just the WETH9 contract deployed at addresses in `@0x/contract-addresses`?

Reply:
`erc20tokenWrapper` and `erc721tokenWrapper` were similar and I just left those out.

Reply:
Yeah, let's do with the WETH contract address in contract-addresses. If they want something else, they can instantiate it themselves.